### PR TITLE
Update command summary of UG

### PR DESCRIPTION
**Pull request by teamotitan**
Previously, command summary of our UG is vague for edit command, and there is no summary for the other commands, hence readability becomes an issue. Displayed parameters for some error messages does not correspond accurately to what was advocated in the UG. 

Let's
* update the command summary of the UG so as to enhance user readability
* Update the parameter names in the various MESSAGE_USAGEs

approved in issue: https://github.com/nus-cs2103-AY2425S1/forum/issues/663

Updated:
image

### CODE REVIEW

**Comment by Feng1231** [comment]
can you include a screenshot?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -54,19 +54,19 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
-**Owe**    | `owe INDEX hr/HOURS_OWED`<br> e.g., `owe 1 hr/1.5`
-**Settle** | `settle INDEX amount/AMOUNT`<br> e.g., `settle 1 amount/500.00`
-**Income**  | `income`
-**Remind**   | `remind`
-**Clear**  | `clear`
-**Exit**   | `exit`
+**Help** : opens up the help page  | `help`
+**List** : lists all students in UGTeach | `list`
+**Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID] [owed/OWED]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
+**Edit** : edits a student in UGTeach | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/SCHEDULE] [s/SUBJECT] [r/RATE] [paid/PAID] [owed/OWED]`<br> e.g.,`edit 2 paid/1200.00 owed/0`
```
Comment (Feng1231) [inline docs/UserGuide.md:60]: PAID_AMOUNT, OWED_AMOUNT

File: docs/UserGuide.md [hunk 2]
```diff
@@ -54,19 +54,19 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
-**Pay**   | `pay INDEX hr/HOURS_PAID`<br> e.g., `pay 1 hr/2.5`
-**Owe**    | `owe INDEX hr/HOURS_OWED`<br> e.g., `owe 1 hr/1.5`
-**Settle** | `settle INDEX amount/AMOUNT`<br> e.g., `settle 1 amount/500.00`
-**Income**  | `income`
-**Remind**   | `remind`
-**Clear**  | `clear`
-**Exit**   | `exit`
+**Help** : opens up the help page  | `help`
+**List** : lists all students in UGTeach | `list`
+**Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID] [owed/OWED]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
```
Comment (Feng1231) [inline docs/UserGuide.md:59]: PAID_AMOUNT, OWED_AMOUNT

**Review by Feng1231 — CHANGES_REQUESTED** [review]
Please change the PAID and OWED

File: docs/UserGuide.md [hunk 3]
```diff
@@ -54,19 +54,19 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
+**Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
+**Edit** : edits a student in UGTeach | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/SCHEDULE] [s/SUBJECT] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`<br> e.g.,`edit 2 paid/1200.00 owed/0`
+**Delete** : deletes a student in UGTeach | `delete INDEX`<br> e.g., `delete 3`
+**Find** : finds a student in UGTeach with part of name matching one of keywords, **and** their schedule falling on one fo the days specified | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
+**Pay** : updates amount paid by a student | `pay INDEX hr/HOURS_PAID`<br> e.g., `pay 1 hr/2.5`
+**Owe** : updates amount owed by a student | `owe INDEX hr/HOURS_OWED`<br> e.g., `owe 1 hr/1.5`
+**Settle** : updates amount paid and owed by student(used when the student repays an amount owed to you) | `settle INDEX amount/AMOUNT`<br> e.g., `settle 1 amount/500.00`
```
Comment (TY1Fan) [inline docs/UserGuide.md:65]: I think there should be a space like `student (used when....)`

File: docs/UserGuide.md [hunk 4]
```diff
@@ -54,19 +54,19 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Find**   | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
-**Pay**   | `pay INDEX hr/HOURS_PAID`<br> e.g., `pay 1 hr/2.5`
-**Owe**    | `owe INDEX hr/HOURS_OWED`<br> e.g., `owe 1 hr/1.5`
-**Settle** | `settle INDEX amount/AMOUNT`<br> e.g., `settle 1 amount/500.00`
-**Income**  | `income`
-**Remind**   | `remind`
-**Clear**  | `clear`
-**Exit**   | `exit`
+**Help** : opens up the help page  | `help`
```
Comment (DennieDan) [inline docs/UserGuide.md:57]: I think this one can change to `opens up the Help window`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -54,19 +54,19 @@ If you can type fast, UGTeach can get your contact management tasks done **faste
...
+**List** : lists all students in UGTeach | `list`
+**Add**  : adds a student to UGTeach  | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS t/SCHEDULE s/SUBJECT r/RATE [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]` <br> e.g., `add n/James Ho p/82224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/Monday-0800-1000 s/GP r/300 paid/300`
+**Edit** : edits a student in UGTeach | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/SCHEDULE] [s/SUBJECT] [r/RATE] [paid/PAID_AMOUNT] [owed/OWED_AMOUNT]`<br> e.g.,`edit 2 paid/1200.00 owed/0`
+**Delete** : deletes a student in UGTeach | `delete INDEX`<br> e.g., `delete 3`
+**Find** : finds a student in UGTeach with part of name matching one of keywords, **and** their schedule falling on one fo the days specified | `find [n/KEYWORD [MORE_KEYWORDS]] [d/DAY [MORE_DAYS]]`<br> e.g., `find n/Alex d/Friday`
```
Comment (DennieDan) [inline docs/UserGuide.md:62]: `falling on one "of" the days`

File: docs/UserGuide.md (hunk 3 above)
Comment (DennieDan) [inline docs/UserGuide.md:65]: can perhaps add a space between `student` and the` bracket

`by student (used...`

**Review by DennieDan — COMMENTED** [review]
LGTM, some minor changes

File: src/main/java/seedu/address/logic/commands/OweCommand.java [hunk 6]
```diff
@@ -34,7 +34,7 @@ public class OweCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the amount the student owes "
             + "by the index number used in the displayed Student list.\n"
             + "Parameters: INDEX (must be a positive integer) "
-            + PREFIX_HOUR + "HOUR OWED (must be a positive multiple of 0.5)\n"
+            + PREFIX_HOUR + "HOUR_OWED (must be a positive multiple of 0.5)\n"
```
Comment (Feng1231) [inline src/main/java/seedu/address/logic/commands/OweCommand.java:37]: HOURS_OWED

**Review by Feng1231 — COMMENTED** [review]
There's still one minor error. Please help to update it

File: docs/UserGuide.md (hunk 3 above)
Comment (teamotitan) [reply docs/UserGuide.md:65]: Tks for that!

File: docs/UserGuide.md (hunk 4 above)
Comment (teamotitan) [reply docs/UserGuide.md:57]: Tyty!

File: docs/UserGuide.md (hunk 5 above)
Comment (teamotitan) [reply docs/UserGuide.md:62]: Tyty!

File: src/main/java/seedu/address/logic/commands/OweCommand.java (hunk 6 above)
Comment (teamotitan) [reply src/main/java/seedu/address/logic/commands/OweCommand.java:37]: Tyty!

Comment (Feng1231) [reply src/main/java/seedu/address/logic/commands/OweCommand.java:37]: This hasn't been changed right?

Comment (teamotitan) [reply src/main/java/seedu/address/logic/commands/OweCommand.java:37]: I changed it to HOURS_OWED?

**Review by Feng1231 — APPROVED** [review]

**Pull request merged**